### PR TITLE
Completed react-form-controls - Ellie Hwang

### PR DESCRIPTION
**Pull request by ellie-hwang**
react-form-controls

### CODE REVIEW

File: react-form-controls/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  handleChange(event) {
+    this.setState({ email: event.target.value });
+  }
+
+  render() {
+    return (
+      <form onSubmit = {this.handleSubmit}>
+        <label htmlFor="email"> Email </label>
+        <input type="text" id="email" onChange={this.handleChange} />
```
Comment (thebearingedge) [inline react-form-controls/src/index.jsx:27]: Be sure to also pass a `value` prop to your input so that it is fully "controlled".

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: react-form-controls/src/index.jsx (hunk 1 above)
Comment (ellie-hwang) [reply react-form-controls/src/index.jsx]: Gotcha, value prop has been added to the input

File: react-form-controls/src/index.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  handleChange(event) {
+    this.setState({ email: event.target.value });
+  }
+
+  render() {
+    return (
+      <form onSubmit = {this.handleSubmit}>
+        <label htmlFor="email"> Email </label>
+        <input type="text" id="email" value={this.state.value} onChange={this.handleChange} />
```
Comment (thebearingedge) [inline react-form-controls/src/index.jsx:27]: You mean to pass `this.state.email`

```jsx
value={this.state.email}
```

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: react-form-controls/src/index.jsx (hunk 2 above)
Comment (ellie-hwang) [reply react-form-controls/src/index.jsx]: Gotcha, it has been revised to {this.state.email}

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**